custom_reports/columns.rst

```diff
@@ -1,13 +1,15 @@
 Columns
 =======
 
-This article documents all available columns in the :ref:`custom-reports-http-api`.
+This article documents columns available in the :ref:`custom-reports-http-api`
+by default. Additional columns may become available through
+:ref:`custom-reports-integrations`.
 
 Metrics
 -------
 
-The table below lists all base metrics that may be used in queries. Additional
-metrics may be created using dimension transformations.
+The table below lists base metrics that may be used in queries by default.
+Additional metrics may be created using dimension transformations.
```

Comment:
Don't we also add metrics via integrations?

Reply:
It's mentioned just above this sentence, no point repeating it.